src/tool/core/copy.rs

```diff
@@ -0,0 +1,66 @@
+use std::{
+    fs,
+    io::{BufRead, BufReader},
+    path::Path,
+};
+
+use anyhow::{Context, Result};
+use arboard::Clipboard;
+use itertools::Itertools;
+use log::{debug, info};
+
+use super::generate::SEPARATOR;
+
+pub(crate) fn copy() -> Result<()> {
+    // Logging
+    info!("Starting copy function");
+
+    // Find the first file in src/ that is not lib.rs
```

Comment:
Given the discussion in the conversation of this PR we need to decide how we want to handle this. One other other option I was thinking about is we have a config file in the user's home directory which tracks the most recent problems they worked on and we could get the most recent one from there. I can help with implementation if this seems daunting.

Reply:
That said the easiest option is to have the user specify the problem but I'm concerned about the UX of that option. I feel auto complete may actually make specifying the file not that hard but they would have to think about what the problem number or name is and this might just end up being harder than just doing the copy and paste themselves.

Reply:
Resolve this in favor of #86?

Reply:
No, Issue https://github.com/rust-practice/cargo-leet/issues/86 is a reminder to document the current structure it doesn't solve our decision regarding how we want to implement this feature. I think we have a few choices to pick from (you're also welcome to add any I may have missed):
1. Ask the user for the filename
2. Ask the user for the problem and determine the filename from that
3. Add the capability to the tool to save setting and track the problems that they recently started